### PR TITLE
test(map): verify `map.jinja` dump using `_mapdata` state

### PR DESCRIPTION
**Pull request by baby-gnu**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [x] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [x] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

Dump `map.jinja` value and verify it against InSpec reference files:

- test(inspec): add InSpec shared resources
- test(inspec): verify `mapdata` dump against reference files
- test(inspec): dump `mapdata` content

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [x] Included in Kitchen (i.e. under `state_top`).
- [x] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

**Comment by baby-gnu** [comment]
I tested locally, only `gentoo` systems are failing but not for the `map.jinja` verification.

The rubocop is failing only for pre-existing code.

File: test/integration/default/controls/_mapdata.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+control 'TEMPLATE._mapdata' do
```
Comment (myii) [inline test/integration/default/controls/_mapdata.rb]:
```suggestion
control 'cert._mapdata' do
```

**Review by myii — APPROVED** [review]
Thanks @baby-gnu.  Checked with the `ssf-formula` and all is fine except the one fix that I've highlighted inline.

Once you've pushed that change, please feel free to merge to prevent any further delay.

File: test/integration/default/controls/_mapdata.rb (hunk 1 above)
Comment (baby-gnu) [reply test/integration/default/controls/_mapdata.rb]: :roll_eyes: I `cp` the directory as a dumb.

Thanks a lot.

**Pull request merged**